src/slave_node/measuring_node.py

```diff
@@ -0,0 +1,35 @@
+import canopen
+import logging
+import yaml
+from time import sleep
+
+'''
+    We are not going to use this for the project but can be
+    used for testing purposes.
+'''
+
+class MeasuringSlaveNode:
+    def __init__(self):
+        with open('../config.yaml', 'r') as ymlfile:
+            self.config = yaml.safe_load(ymlfile)
+        # local node
+        self.connectToNetwork(canopen.Network())
+        self.network.create_node(1, '../eds_files/measuring_distance_node.eds')
+        while True:
+            sleep(1)
+            print(self.network[1].sdo[0x2000].phys)
+            self.network[1].sdo.download(0x2000, 0, bytes([self.network[1].sdo[0x2000].phys + 1, 0]))
+            self.network.sync.transmit()
+
+    def connectToNetwork(self, network):
+        self.network = network
+        try:
+            self.network.connect(bustype=self.config['canopen']['bustype'],
+                                 channel=self.config['canopen']['channel'],
+                                 bitrate=self.config['canopen']['bitrate'])
+        except OSError:
+            logging.error('CanOpenListener is unable to listen to network,'
```

Comment:
misschien hier iets vermelden dat dit een measuring node betreft of wat andere nuttige info? Om debugging makkelijker te maken?

Reply:
Uhm dit gaat om het netwerk niet om nodes ofzo.. Ik zal wel meer nuttige info toevoegen zoals channel bitrate etc